public/app/plugins/panel/geomap/style/utils.test.ts

```diff
@@ -0,0 +1,73 @@
+import { ResourceDimensionMode } from 'app/features/dimensions';
+import { StyleConfig } from './types';
+import { getStyleConfigState } from './utils';
+describe('style utils', () => {
+  it('should fill in default values', async () => {
+    const cfg: StyleConfig = {
+      color: {
+        field: 'Price',
+        fixed: 'dark-green',
+      },
+      opacity: 0.4,
+      size: {
+        field: 'Count',
+        fixed: 5,
+        max: 15,
+        min: 2,
+      },
+      symbol: {
+        fixed: 'img/icons/marker/star.svg',
+        mode: ResourceDimensionMode.Fixed, // 'fixed',
+      },
+      textConfig: {
+        fontSize: 12,
+        offsetX: 0,
+        offsetY: 0,
+        // textAlign: 'center',
+        // textBaseline: 'middle',
```

Comment:
Any reason we aren't including text align / baseline as part of this test?

Reply:
because the typing is off and it is unrelated enough that we can fix in a new PR :)